src/test/java/com/suse/netapi/client/SaltStackClientMockTests.java

```diff
@@ -0,0 +1,111 @@
+package com.suse.saltstack.netapi.client;
+
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.Before;
+
+import static org.junit.Assert.*;
+
+import com.github.restdriver.clientdriver.ClientDriverRule;
+import com.github.restdriver.clientdriver.ClientDriverRequest.Method;
+import com.github.restdriver.clientdriver.ClientDriverRequest;
+
+import static com.github.restdriver.clientdriver.RestClientDriver.giveResponse;
+import static com.github.restdriver.clientdriver.RestClientDriver.onRequestTo;
+
+import com.github.restdriver.clientdriver.exception.ClientDriverFailedExpectationException;
+
+import com.suse.saltstack.netapi.exception.SaltStackException;
```

Comment:
In order to compile this in my IDE I had to add 3 more import statements:

    import com.suse.saltstack.netapi.client.SaltStackClient;
    import com.suse.saltstack.netapi.client.SaltStackRunResults;
    import com.suse.saltstack.netapi.client.SaltStackToken;